app/models/in_person_enrollment.rb

```diff
@@ -0,0 +1,11 @@
+class InPersonEnrollment < ApplicationRecord
+  belongs_to :user
+  belongs_to :profile
+  enum status: {
+    pending: 0,
+    passed: 1,
+    failed: 2,
+    expired: 3,
+    canceled: 4,
+  }
+end
```

Comment:
It occurs to me that we're allowing an InPersonEnrollment to be created with any user and profile; we don't validate that the user is associated with the profile, but they always should be. You could add a custom validator at the top like:
```ruby
  validate :profile_belongs_to_user
```

and then add the private method at the bottom:
```ruby
  private

  def profile_belongs_to_user
    errors.add(:profile, "must belong to user") unless profile&.user == user
  end
```

...and then write a test to make sure it works as expected

Reply:
Added user/profile match validation for `InPersonEnrollment` along with associated test.